doc/using_pytorch.rst

```diff
@@ -566,11 +570,91 @@ The function should return a byte array of data serialized to content_type.
 The default implementation expects ``prediction`` to be a torch.Tensor and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
+
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a tar.gz file.
+* Upload the tar.gz file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``predict_fn`` function that calls the loaded model to get a prediction.
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output. 
+For information about how to write an inference script, see `Serve a PyTorch Model <#serve-a-pytorch-model>`_.
+Save the inference script as ``inference.py`` in the same folder where you saved your PyTorch model. 
+
+Package model artifacts into a tar.gz file
+------------------------------------------
+
+The directory structure where you saved your PyTorch model should look something like the following:
+
+|   my_model
+|           |--model.pth
+|        
+|           code
+|               |--inference.py
+|               |--requirements.txt
+
+Where ``requirments.txt`` is an optional file that specifies dependencies on third-party libraries.
+
+With this file structure, run the following command to package your model as a ``tar.gz`` file:
+
+``tar -czf model.tar.gz my_model code``
+
+Upload model.tar.gz to S3
+-------------------------
+
+After you package your model into a ``tar.gz`` file, upload it to an S3 bucket by running the following python code:
+
+.. code:: python
+
+    import boto3
+    import sagemaker
+    s3 = boto3.client('s3')
+
+    from sagemaker import get_execution_role
+    role = get_execution_role()
+
+    response = s3.upload_file('model.tar.gz', 'my-bucket', '%s/%s' %('my-path', 'model.tar.gz'))
```

Comment:
I think it would be better to show off the Python SDK's S3 helper: https://sagemaker.readthedocs.io/en/stable/s3.html#sagemaker.s3.S3Uploader

```suggestion
    from sagemaker.s3 import S3Uploader

    S3Uploader.upload('model.tar.gz', 's3://my-bucket/my-path/model.tar.gz')
```